guides/doc-Provisioning_Guide/topics/proc_building-a-discovery-image.adoc

```diff
@@ -0,0 +1,119 @@
+[[building-a-discovery-image]]
+= Building a Discovery Image
+
+The Discovery image is a minimal operating system based on {RHEL} that is PXE-booted on hosts to acquire initial hardware information and to check in to the Satellite Server. Discovered hosts keep running the Satellite Discovery image until they are rebooted into Anaconda, which then initiates the provisioning process. The `foreman-discovery-image` package contains this image. You must install the package on the {SmartProxy} that provides TFTP services.
+
+Use this procedure to build a {Project} discovery image or rebuild an image if you change configuration files.
+
+Do not use this procedure on your production {Project} or {SmartProxy}.
+
+.Prerequisites
+
+Install the `livecd-tools` package:
+ifeval::["{build}" == "foreman"]
+----
+# yum install livecd-tools
+----
+endif::[]
+ifeval::["{build}" == "satellite"]
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-maintain} packages install livecd-tools
+----
+endif::[]
+
+Because Anaconda installer cannot publish through HTTPS, you must enable publishing through HTTP for Kickstart repositories:
+
+. In the {Project} web UI, navigate to *Content* > *Products* and in the *Products* window, click the *Repositories* tab.
+. Select a Kickstart repository and for the *Publish via HTTP*, option, click the *Edit* icon, select the check box, and click *Save*.
+. Repeat the previous steps for the {Project} repository.
+
+Note that publishing via HTTP does not apply to any Red{nbsp}Hat repositories.
+
+.Procedure
+
+To build the {Project} discovery image, complete the following steps:
+
+. Open the `/usr/share/foreman-discovery-image/foreman-discovery-image.ks` file for editing:
++
+[options="nowrap" subs="+quotes"]
+----
+# vim /usr/share/foreman-discovery-image/foreman-discovery-image.ks
+----
++
+. Replace `repo --name=rhel --baseurl=http://download/00000` with with your own repos for RHEL and {Project} with your repository URLs. To find the URLs, navigate to *Content* > *Products* and click the *Repositories* tab and copy the URL for both repositories into the file:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+repo --name=rhel --baseurl=http://download/released/RHEL-7/7.4/Server/x86_64/os/
+repo --name=sat --baseurl=http://download2/nightly/Satellite/{ProductVersion}/candidate/latest-Satellite-{ProductVersion}-RHEL-7/compose/Satellite/x86_64/os/
```

Comment:
Replace this with:

If foreman:

```
repo --name=centos --mirrorlist=http://mirrorlist.centos.org/?release=7&arch=$basearch&repo=os
repo --name=centos-updates --mirrorlist=http://mirrorlist.centos.org/?release=7&arch=$basearch&repo=updates
repo --name=epel7 --mirrorlist=https://mirrors.fedoraproject.org/metalink?repo=epel-7&arch=$basearch
repo --name=centos-sclo-rh --mirrorlist=http://mirrorlist.centos.org/?release=7&arch=x86_64&repo=sclo-rh
repo --name=foreman-rails-el7 --baseurl=https://yum.theforeman.org/rails/foreman-nightly/el7/$basearch/
repo --name=foreman-el7 --baseurl=http://yum.theforeman.org/nightly/el7/$basearch/
repo --name=foreman-plugins-el7 --baseurl=http://yum.theforeman.org/plugins/nightly/el7/$basearch/
```

If Satellite:

```
repo --name=rhel --baseurl=http://{foreman_example_com}/pulp/repos/MyOrg/Library/content/dist/rhel/server/7/7.7/x86_64/kickstart/
repo --name=sat --baseurl=http://{foreman_example_com}/pulp/repos/MyOrg/Library/content/dist/rhel/server/7/7Server/x86_64/sat-capsule/6.7/os/
```

Also we need to add a note (satellite only) that both "Red Hat Enterprise Linux 7 Server Kickstart x86_64 7.7" and "Red Hat Satellite Capsule 6.7 for RHEL 7 Server RPMs x86_64" must be enabled and synced in order to use them in the build process.